cosmopolitan/management/commands/_naturalearthdata.py

```diff
@@ -62,11 +62,11 @@ def process_cities():
     Polygon.objects.filter(type='city').delete()
 
     for feature in data["features"]:
-        json_city_name = feature["properties"]["GN_ASCII"]
+        json_city_id = int(feature["properties"]["GEONAMEID"])
         try:
-            if json_city_name is None:
+            if json_city_id is None:
                 continue
-            city = City.objects.get(name=json_city_name)
+            city = City.objects.get(id=str(json_city_id))
```

Comment:
Django cities IDs are equal to GeoNames IDs and defined as integers on the DB. 
Cosmopolitan cities IDs are now populated by this ID instead of the slug. 
I kept cosmopolitan city model id column as charfield, as I was afraid of breaking something. 
GeoNameID in NaturalEarth data comes in decimal form for some reason I am not aware of. 
Would it be more appropriate if the type casting was on just one line? Or there is a better way? Could I change cosmopolitan's city model id column to IntegerField? I am not so fluent in Python.